os/arch/arm/src/stm32l4/stm32l4_ospi.c

```diff
@@ -1920,9 +1920,7 @@ FAR struct mtd_dev_s *up_flashinitialize(void)
     priv->bwrite = this_bwrite;
     priv->read = this_read;
     priv->ioctl = this_ioctl;
-#ifdef CONFIG_MTD_REGISTRATION
-    priv->mtd.name = "stm32l4_flash";
-#endif
```

Comment:
We should not just simply remove the line. If you check mtd_dev_s structure you can see that it was changed to ```priv->name``` instead of ```priv->mtd.name```. 

https://github.com/Samsung/TizenRT/blob/436a98058060741956e3b7bc347f96553549bcd3/os/include/tinyara/fs/mtd.h#L168